fsw/src/to_lab_app.c

```diff
@@ -251,6 +257,28 @@ int32 TO_LAB_EnableOutput(const TO_LAB_EnableOutputCmd_t *data)
         TO_LAB_Global.downlink_on = true;
     }
 
+    /* 
+        Start hostname snippet from: https://stackoverflow.com/questions/38002016/problems-with-gethostbyname-c
+    */
+    struct hostent *he;
+	struct in_addr **addr_list;
+    int i;
+
+    if ( (he = gethostbyname(TO_LAB_Global.tlm_dest_IP) ) != NULL) 
+    {
+        addr_list = (struct in_addr **) he->h_addr_list;
+        for(i = 0; addr_list[i] != NULL; i++) 
+        {
+            //Return the first one;
```

Comment:
This code actually returns the last one, not the first one.  Why not return the first one?  It is computationally cheaper and less code... replace the entire for block with
strcpy(TO_LAB_Global.tlm_dest_IP, inet_ntoa(*addr_list[0]));

Reply:
I think this was originally to get the additional null check, realistically a single `if (addr_list[0] != NULL)` though should do it! Good catch.